test/e2e/predictor/test_multi_model_serving.py

```diff
@@ -0,0 +1,222 @@
+# Copyright 2021 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from kubernetes import client
+from kfserving import (
+    constants,
+    KFServingClient,
+    V1beta1PredictorSpec,
+    V1alpha1TrainedModel,
+    V1alpha2EndpointSpec,
+    V1beta1InferenceService,
+    V1beta1InferenceServiceSpec,
+    V1alpha1ModelSpec,
+    V1alpha1TrainedModelSpec,
+    V1beta1SKLearnSpec,
+    V1beta1XGBoostSpec,
+    V1alpha2TritonSpec,
+)
+
+from ..common.utils import predict, get_cluster_ip
+from ..common.utils import KFSERVING_TEST_NAMESPACE
+
+api_v1alpha1_version = constants.KFSERVING_GROUP + "/v1alpha1"
+api_v1beta1_version = constants.KFSERVING_GROUP + "/" + constants.KFSERVING_V1BETA1_VERSION
+
+KFServing = KFServingClient(config_file=os.environ.get("KUBECONFIG", "~/.kube/config"))
+
+def test_mms_sklearn_kfserving():
+    # Define an inference service
+    predictor = V1beta1PredictorSpec(
+        min_replicas=1,
+        sklearn=V1beta1SKLearnSpec(
+            protocol_version="v1",
+            resources=client.V1ResourceRequirements(
+                requests={"cpu": "100m", "memory": "256Mi"},
+                limits={"cpu": "100m", "memory": "256Mi"},
+            )
+        )
+    )
+
+    service_name = "isvc-sklearn"
```

Comment:
probably name to `isvc-sklearn-mms` otherwise this may cause conflicts with the single model test.